examples/index.html

```diff
@@ -1,121 +1,118 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Leaflet.DistortableImage Example</title>
-    <meta charset="utf-8" />
 
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<head>
+  <title>Leaflet.DistortableImage Example</title>
+  <meta charset="utf-8" />
 
-    <script
-      src="../node_modules/leaflet/dist/leaflet.js"
-      type="text/javascript"
-      charset="utf-8"
-    ></script>
-    <link
-      href="../node_modules/font-awesome/css/font-awesome.min.css"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="../node_modules/leaflet/dist/leaflet.css"
-      type="text/css"
-      media="screen"
-      title="no title"
-      charset="utf-8"
-    />
-    <script src="../node_modules/leaflet-toolbar/dist/leaflet.toolbar.js"></script>
-    <link
-      href="../node_modules/leaflet-toolbar/dist/leaflet.toolbar.css"
-      rel="stylesheet"
-    />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <!-- for full-res export -->
-    <script src="../node_modules/jquery/dist/jquery.js"></script>
-    <script src="../node_modules/webgl-distort/dist/webgl-distort.js"></script>
-    <script src="../node_modules/glfx/glfx.js"></script>
+  <script src="../node_modules/leaflet/dist/leaflet.js" type="text/javascript" charset="utf-8"></script>
+  <link href="../node_modules/font-awesome/css/font-awesome.min.css" rel="stylesheet" />
+  <link rel="stylesheet" href="../node_modules/leaflet/dist/leaflet.css" type="text/css" media="screen" title="no title"
+    charset="utf-8" />
+  <script src="../node_modules/leaflet-toolbar/dist/leaflet.toolbar.js"></script>
+  <link href="../node_modules/leaflet-toolbar/dist/leaflet.toolbar.css" rel="stylesheet" />
+  <!-- for pattern-mining utility -->
+  <script src="../node_modules/matcher-core/orb.core.com.js"></script>
```

Comment:
Maybe we ought to move this to a unique example file called "matcher"?

Reply:
@rexagod i think we'd better follow up on this, `matcher.html` sounds like a good name for the example!

Reply:
Sounds good, on it! 👍